### ISSUE TITLE
Any examples about MFConv?

### ISSUE THREAD

**Issue by MagicPF** [rusty1s/pytorch_geometric#3063]
Dear developers,

Could you provide some examples about using MFConv please?

Best regards,
MagicPF

**Comment by rusty1s** [comment]
I'm sorry that we do not have any examples for `MFConv`. However, it's use is similar to any other operator included in PyG. Do you have any problems with applying `MFConv` to your specific use-case?

**Comment by MagicPF** [comment]
Do you have some examples that create a dataset with labels, which is accessible for the MFConv or other Convs?

**Comment by rusty1s** [comment]
A small example for utilizing `MFConv` on the `ZINC` dataset:
```python
import torch
from torch_geometric.nn import MFConv, global_add_pool
from torch_geometric.data import DataLoader
from torch_geometric.datasets import ZINC

dataset = ZINC('/tmp/ZINC')
loader = DataLoader(dataset, batch_size=32, shuffle=True)


class GNN(torch.nn.Module):
    def __init__(self, hidden_channels, out_channels, num_layers):
        super().__init__()

        self.emb = torch.nn.Embedding(21, hidden_channels)
        self.convs = torch.nn.ModuleList()
        for _ in range(num_layers):
            self.convs.append(MFConv(hidden_channels, hidden_channels))
        self.lin = torch.nn.Linear(hidden_channels, 1)

    def forward(self, x, edge_index, batch):
        x = self.emb(x.squeeze(-1))
        for conv in self.convs:
            x = conv(x, edge_index).relu()
        x = global_add_pool(x, batch)
        return self.lin(x)


model = GNN(64, 1, num_layers=3)

for data in loader:
    out = model(data.x, data.edge_index, data.batch)
    print(out.shape)
```
If you show me from which data you want to create a dataset, I can give you more guidance on how to do so.

**Comment by MagicPF** [comment]
The format of my dataset is a set of SMILES formula with a 0/1 labels.

**Comment by rusty1s** [comment]
[Here](https://github.com/rusty1s/pytorch_geometric/blob/master/torch_geometric/datasets/molecule_net.py) is an example on how to convert SMILES strings to graph data.
